### ISSUE TITLE
fa-qq need some eyes and maybe a belly

### ISSUE THREAD

**Issue by francisfeng** [FortAwesome/Font-Awesome#6906]
The current "qq" (logo of Tencent Inc.) just doesn't look right. It's merely a shape of penguin. Adding two eyes would dramatically improve its legibility. Adding a belly does that too.

**Comment by tagliala** [comment]
The icon is taken from here: http://www.imqq.com/

It seems the official icon

**Comment by francisfeng** [comment]
Yes indeed, it's the official icon. But wouldn't be nicer if the penguin has two eyes and a belly? It's just too hard to recognize at first glance, even for a native Chinese who has been along with the logo for more than ten years.

**Comment by sensibleworld** [comment]
Doing a little management in Github and (batch) closing out potentially stale issues. If the above still needs attention, please let us know and we’ll re-open!
